services/s3.py

```diff
@@ -2,64 +2,209 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("s3")
+sts_client = boto3.client("sts")
+s3control_client = boto3.client("s3control")
+backup_client = boto3.client("backup")
 
 
 def s3_access_point_in_vpc_only():
+    compliant_resources = []
+    non_compliant_resources = []
+    account_id = sts_client.get_caller_identity().get("Account")
+    access_points = s3control_client.list_access_points(AccountId=account_id)["AccessPointList"]
+
+    for access_point in access_points:
+        if access_point["NetworkOrigin"] == "VPC":
+            compliant_resources.append(access_point["AccessPointArn"])
+        else:
+            non_compliant_resources.append(access_point["AccessPointArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def s3_bucket_default_lock_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    buckets = client.list_buckets()["Buckets"]
+
+    for bucket in buckets:
+        try:
+            response = client.get_object_lock_configuration(Bucket=bucket["Name"])
+            compliant_resources.append(f"arn:aws:s3:::{bucket['Name']}")
+        except Exception as e:
+            if e.__class__.__name__ == "ObjectLockConfigurationNotFoundError":
```

Comment:
Exception의 class가 `botocore.exceptions.ClientError`이기 때문에 예외 처리 방법이 다른 방법으로 구현되어야 합니다.
```suggestion
        except botocore.exceptions.ClientError as e:
            if e.response['Error']['Code'] == "ObjectLockConfigurationNotFoundError":
```